automation_tools/satellite6/hammer.py

```diff
@@ -386,3 +386,21 @@ def attach_subscription_to_host_from_satellite(
         organization_id, product_name)
     return hammer('host subscription attach --subscription-id {0} '
                   '--host {1}'.format(subscription_id, hostname))
+
+
+def hammer_determine_cv_and_env_from_ak(ak_name, organization_id):
```

Comment:
Its returning both cv and lenv. Both keys 'cv' and 'lenv' are being set to dict and dict is getting returned. 
Did you misunderstood ?? :)